make/jdk/src/classes/build/tools/classlist/HelloClasslist.java

```diff
@@ -160,13 +168,26 @@ record B(int b) { }
             case B b -> b.b;
             default -> 17;
         };
+        // record run-time methods
+        o.equals(new B(5));
+        o.hashCode();
         LOGGER.log(Level.FINE, "Value: " + value);
 
         // The Striped64$Cell is loaded rarely only when there's a contention among
         // multiple threads performing LongAdder.increment(). This results in
         // an inconsistency in the classlist between builds (see JDK-8295951).
         // To avoid the problem, load the class explicitly.
         Class<?> striped64Class = Class.forName("java.util.concurrent.atomic.Striped64$Cell");
+
+        // Initialize FFM linkers
+        System.loadLibrary("java");
+        Linker linker = Linker.nativeLinker();
+        SymbolLookup lookup = SymbolLookup.loaderLookup();
+        var signature = FunctionDescriptor.of(ADDRESS,
+                ADDRESS, ADDRESS, ADDRESS,
+                JAVA_BOOLEAN, ADDRESS, ADDRESS);
+        Optional<MemorySegment> symbol = lookup.find("Java_java_lang_Class_forName0");
+        var _ = linker.downcallHandle(symbol.orElseThrow(), signature);
```

Comment:
FWIW, passing just the signature should result in mostly the same code being generated. The address is just attached using MethodHandles::insertArguments.

Reply:
So just `Linker.nativeLinker().downcallHandle(MemorySegment.ofArray(new byte[0]), signature)` suffices?

Reply:
Yes, that would work, but I was thinking just `Linker.nativeLinker().downcallHandle(signature)`